### DISCUSSION TITLE
I can't download GitHub Classroom roster

### DISCUSSION THREAD

**Discussion by caburu** [community/community#61148] [GitHub Education]
### Hi everyone,

I'm trying to download a Classroom's roster but when I click in the option nothing happens.

I tried in this way:
- Access the Classroom page
- Students tab
- Download button
- In the showed box none of the options works (all students neither grouping). Simply nothing happens.

Any ideas?

Thanks!

**Comment by ssardina** [comment]
Same here, tried different browser. This is super important and we need to get the list of students with their links to their accounts

**Reply by ssardina** [reply]
@caburu have you found any work-around? I have tried firefox, chrome, brave and safari. All the same, clicking does nothing.

I cannot get the roster which is critical...

**Reply by nasuoa** [reply]
Same here, tried in Chrome and Firefox. There's no reaction from clicking the button.

**Reply by ssardina** [reply]
Gee this is very painful.. Do you know if developers look at these posts frequently?

**Reply by nasuoa** [reply]
Not sure, but I think the following Discussion forum gets more exposure when it comes to GH Classroom matters, so I have posted it there and linked back here:

https://github.com/community/Global-Campus-Teachers/discussions/241

**Reply by ssardina** [reply]
Thanks, I followed it up

**Comment by caburu** [comment]
> @caburu have you found any work-around? I have tried firefox, chrome, brave and safari. All the same, clicking does nothing.
> 
> I cannot get the roster which is critical...

No :(
